package-lock.json

```diff
@@ -1205,7 +1205,7 @@
 			"dependencies": {
 				"assign-symbols": {
 					"version": "0.1.1",
-					"resolved": "https://registry.npmjs.org/assign-symbols/-/assign-symbols-0.1.1.tgz",
+					"resolved": "http://registry.npmjs.org/assign-symbols/-/assign-symbols-0.1.1.tgz",
```

Comment:
why are you replacing https with http?

Reply:
Yes, by accident. It's automatic updates, I haven't touched it manually

Reply:
Are you using npm 6.5.0?

Reply:
No, 6.4.1. I'll upgrade it
--